WooCommerce/Classes/POS/Presentation/PointOfSaleCollectCashView.swift

```diff
@@ -70,16 +65,7 @@ struct PointOfSaleCollectCashView: View {
 
             Button(action: {
                 Task { @MainActor in
-                    guard validateAmountOnSubmit() else {
-                        return
-                    }
-                    isLoading = true
-                    do {
-                        try await markComplete()
-                    } catch {
-                        errorMessage = Localization.failedToCollectCashPayment
-                    }
-                    isLoading = false
+                    await submitCashAmount()
```

Comment:
Right now the sizes of some elements become smaller when we increase the text size. If they don't fit anymore, we can just limit their size. In the case of button and text field.

https://github.com/user-attachments/assets/be7ee068-679f-43e9-8972-66f3587ae247